capstone-backend/src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -130,11 +130,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         return;
       }
     } else if (request.getParameter("searchTerm") != null) {
-      volumes.addAll(convertResponseToVolumeData(fullOutput));
-      if (volumes.size() == 0) {
-        response.sendError(HttpServletResponse.SC_NOT_FOUND);
+      // Check for empty response
+      Collection<VolumeData> volumesRetrieved = convertResponseToVolumeData(fullOutput);
+      if (volumesRetrieved == null || volumesRetrieved.size() == 0) {
+        response.sendError(HttpServletResponse.SC_NO_CONTENT);
         return;
       }
+      volumes.addAll(convertResponseToVolumeData(fullOutput));
```

Comment:
nit: Can you just add `volumesRetrieved` here?

Reply:
Yup! Good idea, I created the variable and replaced it in the first if check but not this one